pkg/annotation/capability_test.go

```diff
@@ -115,19 +115,19 @@ func TestParseCapability(t *testing.T) {
 		a = 1
 		b = 2
 	}`,
-			want: &Capability{
+			want: []*Capability{{
 				Name: "thing",
 				Directives: Directives{
 					"key1": map[string]interface{}{"a": int64(1), "b": int64(2)},
 				},
-			},
+			}},
```

Comment:
I would expect additional test cases here to cover where multiple capabilities are returned